rc/core/lint.kak

```diff
@@ -1,6 +1,6 @@
 # your linter should output in this format:
 # {filename}:{line}:{column}: {kind}: {message}
-decl str linter
+decl str lintercmd
```

Comment:
This option would also contain flags and arguments, so `linter_options` can be removed now.